pkg/cmd/workspace/util/creation_data.go

```diff
@@ -29,16 +29,25 @@ func GetCreationDataFromPrompt(config CreateDataPromptConfig) (string, []apiclie
 	var providerRepo *apiclient.GitRepository
 	var err error
 	var workspaceName string
+	// Below 3 maps keeps track of visited repos, their respective namespaces and git providers.
+	// During workspace creation, Lookups will help in avoiding duplicated repo entries and disabling specific
+	// namespaces and git providers list options from further selection under which all repos are visited.
+	// The hierarchy is :
+	// Git Provider ----> Namespaces (organizations/projects) ----> Repositories
+	// Git Provider ----> Repositories (if no orgs available)
+	selectedRepos := make(map[string]bool)
+	selectedReposNamespaces := make(map[string]bool)
```

Comment:
I suggest renaming 
``` golang
selectedReposNamespaces
selectedReposGitProviders
```

to:
```
disabledNamespaces
disabledGitProviders
```

because `selected` seems a bit awkward in this context.

Reply:
agreed, done..